src/compas_xr/assemblies/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .assemblies import AssemblyAssistant
+from .assembly_extensions import AssemblyExtensions
```

Comment:
It's (marginally) preferable to use absolute imports even in the cases in which a relative import would work, so,
```suggestion
from compas_xr.assemblies import AssemblyAssistant
from compas_xr.assembly_extensions import AssemblyExtensions
```